doc/cephadm/services/mgmt-gateway.rst

```diff
@@ -49,6 +49,54 @@ monitoring `mgmt-gateway` takes care of handling HA when several instances of Pr
 available. The reverse proxy will automatically detect healthy instances and use them to process user requests.
 
 
+High Availability for mgmt-gateway service
+==========================================
+
+In addition to provide high availability for the underlying backend services, the mgmt-gateway
+service itself can be configured for high availability, ensuring that the system remains resilient
+even if certain core components for the service fails.
+
+Multiple mgmt-gateway instances can be deployed in an active/standby configuration using keepalived
+for seamless failover. The `oauth2-proxy` service can also be deployed as multiple stateless instances,
+with nginx acting as a load balancer across them using round-robin load balancing. This setup removes
```

Comment:
`can also be` the also here disrupts the flow of the sentence and seems unnecessary.
Maybe: "... can be deployed using multiple stateless instances" works better?

Reply:
I adjusted the sentence :+1: